operations/rotate-linux-windows-passwords/README.md

```diff
@@ -0,0 +1,102 @@
+# Simple Password Rotation with HashiCorp Vault
+This guide demonstrates an automated password rotation workflow using Vault and a simple Bash script. These scripts could be run in a cron job or scheduled task to dynamically update local system passwords on a regular basis.
+
+NOTE: This is *not* the be-all and end-all of password rotation. It is also not a PAM tool. It can do the following:
+
+* Rotate local system passwords on a regular basis
+* Allow systems to rotate their own passwords
+* Store login credentials securely in Vault
+* Ensure that passwords meet complexity requirements
+* Require users to check credentials out of Vault 
+
+In this demo you'll have three terminal windows open. One for running Vault, a second for entering Vault CLI commands, and one for your remote Linux server. The scripts run from the remote GCP instance and connect to Vault to update their stored credentials.
+
+## Prerequisites
+* Vault installed on your local machine: https://www.vaultproject.io/downloads.html
+* GCP account and SDK command line tools installed: https://cloud.google.com/sdk/
+
+## Estimated Time to Complete
+10 minutes
+
+### Step 1: Configure Vault on your local machine
+In a new terminal, run the following commands to get Vault setup on your laptop:
+
+```
+vault server -dev -dev-root-token-id=root
+```
+
+Leave Vault running in this terminal. You can point out API actions as they are logged, such as revoked leases, etc.
+
+### Step 2: Configure a policy
+Open the Vault UI at http://127.0.0.1:8200 and create an ACL policy called `rotate-linux` with the following content:
+
+```
```

Comment:
Would be great if we could include the actual command to write the policy so they can copy/paste 

For example...

```
(cat <<POLICY
path "secret/data/linux/*" {
  capabilities = ["create", "read", "update", "list"]
}
POLICY
) | vault policy write rotate-linux -
```

Reply:
I don't like HEREDOCS.  Especially in a beginner tutorial. 

The right way to do this is save the policy in a file, then run `vault policy write` with the file as input.

I'll include the policy in the source code and the correct command to use.

Reply:
Fair enough, that sounds like a good compromise, just have them run the Vault command for against a policy that's already sitting in the repo directory.